packages/calcite-components/src/components/tabs/tabs.scss

```diff
@@ -1,10 +1,22 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tabs-background-color: The background color of the component.
+ * @prop --calcite-tabs-border-color: The border color of the component.
+ */
+
 :host {
   @apply flex flex-col;
 }
 
 :host([bordered]) {
+  --calcite-tabs-background-color: var(--calcite-color-foreground-1);
```

Comment:
I think these should be on the host and not `:host([bordered]` right?

Reply:
I think background color is only set on bordered.

Reply:
That's right, made a blunder.